.github/workflows/ci.yml

```
@@ -9,92 +9,65 @@
 name: ci
 
 on:
+  push:
+    branches: [ develop ]
   pull_request:
-    branches:
-      - 'master'
-      - 'release/*'
+    branches: [ develop ]
```

GH actions for PRs targeting these branches:

```suggestion
    branches:
      - master
      - release/*
```